package.json

```diff
@@ -11,6 +11,7 @@
     "build": "node ./internals/prod-web.js",
     "build:ext": "node ./internals/prod-ext.js",
     "clean": "rm -rf build/ .parcel-cache",
+    "clean:deep": "yarn clean; rm -rf node_modules",
```

Comment:
When does someone need to delete whole node_modules?? Did you mean node_modules/.cache?

Reply:
Whenever I want to remove all build artifacts. It can be restored by a simple yarn command, and part of .gitignore, so its not really part of the codebase..

Reply:
Build artifacts are in build/, build-ext/, and build-dev/

Sounds like you're trying to do `git clean -fdx`, but I still don't know why

Reply:
> Sounds like you're trying to do `git clean -fdx`, but I still don't know why

Almost, but not _exactly_. I'm trying to reduce the directory to the smallest possible size, without any chance of losing any real (ie. non-derived) information.
 - The current `yarn clean` doesn't do that, because it leaves the `node_modules` directory lying around, which is currently 780MB in my case.
 - The proposed `git clean -fdx` _also_ doesn't do that, because it will remove untracked files, which may very well be new code.

So I still think there is valid use-case for this.